### ISSUE TITLE
FIXME: cmd + , does not open Rimsort settings on MacOS

### ISSUE THREAD

**Issue by Limfjorden** [RimSort/RimSort#218]
**Describe the bug**
Unlike in previous versions, nothing happens when pressing cmd + , . It used to open the settings, like that shortcut does in many other applications. The functionality should return imo.

**To Reproduce**
1. Press cmd + ,
2. Nothing happens

**Expected behavior**
The settings should open.

**Please complete the following information:**
 - OS: Mac OS Ventura 13.5.2
 - RimSort version: Alpha-v1.0.6.2-hf [Edge 4a77934]
 - Please provide a `RimSort.log`: not needed
 - Are you running from Python interpreter or from a compiled RimSort release?
 Release

**Additional context**

**Comment by jefferyharrell** [comment]
Good catch. I believe this is a very easy fix. I'll submit it for inclusion in the next build.
